frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -842,15 +908,15 @@ export class HtmlDocument extends React.Component {
             onClickValidateBtn={this.handleClickSaveDocument}
             version={state.content.number}
             lastVersion={state.timeline.filter(t => t.timelineType === 'revision').length}
-            text={state.content.raw_content}
+            text={displayTranslatedText ? state.translatedRawContent : state.content.raw_content}
             onChangeText={this.handleChangeText}
             isArchived={state.content.is_archived}
             isDeleted={state.content.is_deleted}
             isDeprecated={state.content.status === state.config.availableStatuses[3].slug}
             deprecatedStatus={state.config.availableStatuses[3]}
             isDraftAvailable={state.mode === APP_FEATURE_MODE.VIEW && state.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id && getLocalStorageItem(state.appName, state.content, LOCAL_STORAGE_FIELD.RAW_CONTENT)}
-            onClickRestoreArchived={this.handleClickRestoreArchive}
-            onClickRestoreDeleted={this.handleClickRestoreDelete}
+            // onClickRestoreArchived={this.handleClickRestoreArchive}
+            // onClickRestoreDeleted={this.handleClickRestoreDelete}
```

Comment:
onClickRestoreArchived is indeed unused but onClickRestoreDeleted is and should be kept

Reply:
Corrected.